### PR TITLE
Propose LitmusChaos for Incubation in CNCF

### PR DESCRIPTION
**Pull request by umamukkara**
Hi TOC members,

LitmusChaos project has seen a great amount of traction, adoption and community growth. We as maintainers feel that the project meets the criteria for Incubation stage. We also expressed this intent to the SIG chairs of Sig-App-Delivery and received a go ahead for this next step.

We request you consider this PR as a proposal for LitmusChaos incubation. 

Maintainers
- Uma Mukkara (MayaData)
- Karthik S (MayaData)
- Sumit Nagal (Intuit)
- Jayeshkumar Tank (AWS)
- Maria Kotlyarevskaya (Wrike)

### CODE REVIEW

File: proposals/incubation/LitmusChaos.md [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+- [RedHat](https://www.redhat.com)
+- [Akridata](https://www.akridata.com/)
+- [Workday](https://www.workday.com/)
+- [Anuta Networks](https://www.anutanetworks.com/)
+- [Argo AI](https://www.argo.ai/)
+- [Nokia](https://nokia.com)
+- [AWS](https://aws.amazon.com)
+- [Aicumen Technologies](https://www.aicumen.com/)
+- [ANZ](https://www.anz.com.au/)
+- [QVC](https://www.qvc.com)
```
Comment (xkbarkar) [inline proposals/incubation/LitmusChaos.md:164]: Netapp  (https://www.netapp.com/) also uses litmus in testing/qa

**Comment by xkbarkar** [comment]
This looks good.

**Comment by umamukkara** [comment]
Thank you @amye for suggesting SIG review and recommendation  - We would like to do the formal presentation to SIG-APP-DELIVERY chairs. Can I work with them and schedule?

**Comment by amye** [comment]
Yes! They meet twice a month, March 3rd is their next meeting!

**Comment by umamukkara** [comment]
We presented to Sig-App-Delivery chairs on 3rd March 2021. The presentation slides are here https://docs.google.com/presentation/d/1jir2aHrfkXUsOxhObyjxgNcWy0eZ2XPlaFhDD8i9FWc/edit?usp=sharing

**Comment by umamukkara** [comment]
We already presented at the SIG. Based on the the comments from @lizrice after this week's TOC meeting, our understanding is that the project has to seek the TOC sponsor or TOC itself will assign a sponsor for the project directly. 

Can we request guidance on the next steps / a TOC sponsor?

**Comment by amye** [comment]
> We already presented at the SIG. Based on the the comments from @lizrice after this week's TOC meeting, our understanding is that the project has to seek the TOC sponsor or TOC itself will assign a sponsor for the project directly.
> 
> Can we request guidance on the next steps / a TOC sponsor?

That's a likely *future* process change, not a current change. At the moment, we're waiting on the SIG's recommendations, and I'll raise this with the TOC that you're looking for a sponsor.

**Comment by umamukkara** [comment]
Link to the Due Diligence doc :  [DD doc ](https://docs.google.com/document/d/1_8svqwzs1xYRHU92Qfd1I6UUXYyb8J3Zlpemg2-n4SU/edit?usp=sharing)

**Comment by umamukkara** [comment]
Updated the PR with latest stats and two more end users for production usage: Orange and Anuta Networks.

**Comment by umamukkara** [comment]
Annual review PR: https://github.com/cncf/toc/pull/686

**Comment by dims** [comment]
thanks @caniszczyk ! will review with my sponsor hat on

**Comment by caniszczyk** [comment]
TOC approved! https://lists.cncf.io/g/cncf-toc/message/6483

+1 Binding
9/11
Davanum Srinivas: https://lists.cncf.io/g/cncf-toc/message/6351
Lei Zhang: https://lists.cncf.io/g/cncf-toc/message/6351
Richardo Rocha: https://lists.cncf.io/g/cncf-toc/message/6387
Alena Prokharchyk: https://lists.cncf.io/g/cncf-toc/message/6390
Justin Cormack: https://lists.cncf.io/g/cncf-toc/message/6403
Dave Zolotusky: https://lists.cncf.io/g/cncf-toc/message/6405
Liz Rice: https://lists.cncf.io/g/cncf-toc/message/6406s
Erin Boyd: https://lists.cncf.io/g/cncf-toc/message/6414
Saad Ali: https://lists.cncf.io/g/cncf-toc/message/6417
Cornelia Davis: https://lists.cncf.io/g/cncf-toc/message/6419

**Pull request merged**